packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -109,8 +109,8 @@ impl Abigen {
                     impl #name {
                         #contract_functions
 
-                        pub fn _get_contract_id(&self) -> &Bech32ContractId {
-                            &self.contract_id
+                        pub fn _get_contract_id(&self) -> Bech32ContractId {
+                            self.contract_id.clone()
```

Comment:
Do we need to return an owned value? IMO its better to stick to the ref and push the decision to clone on the user.